{
  "Header": {
    "Original": "CVE-2024-45231",
    "ID": "CVE-2024-45231"
  },
  "Annotations": [
    {
      "Original": "- python-django 3:4.2.16-1",
      "Type": "package",
      "Package": "python-django",
      "Kind": "fixed",
      "Version": "3:4.2.16-1"
    },
    {
      "Original": "NOTE: https://www.djangoproject.com/weblog/2024/sep/03/security-releases/",
      "Type": "NOTE",
      "Description": "https://www.djangoproject.com/weblog/2024/sep/03/security-releases/"
    },
    {
      "Original": "NOTE: https://github.com/django/django/commit/bf4888d317ba4506d091eeac6e8b4f1fcc731199 (4.2.16)",
      "Type": "NOTE",
      "Description": "https://github.com/django/django/commit/bf4888d317ba4506d091eeac6e8b4f1fcc731199 (4.2.16)"
    }
  ]
}